docker/mnt/run.sh

```diff
@@ -0,0 +1,121 @@
+#! /bin/bash
+set -e
+
+if [ -z "$1" ]; then
+    # the main container need not stay running for services
+    exit 0
+fi
+
+function setup() {
+    [ -n "$1" ] && TEST="$1"
+
+    if [[ "$TEST" =~ ^python ]]; then
+
+        scripts/uninstall-requirements.sh
+        pip install \
+            -r requirements/requirements.txt \
+            -r requirements/dev-requirements.txt \
+            coveralls
```

Comment:
Doing this here vs during docker build means that you'll have to do it for every run. That's fine for travis but if you want to use it locally for anything then each run is going to be slower.